### ISSUE TITLE
debug html runnable should be built on every phet-io build

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#674]
I'm not sure about the status of dev, rc, and production builds, but currently the local `grunt --brands=phet-io` command doesn't automatically build the _debug.html file with assertions in. I thought we had decided to make that happen when designing chipper 2.0, but I could be wrong. Currently I'm about 90% sure we will end up wanting debug versions of the sim (with assertions left in) in ever phet-io build ever to support phet-io client devs of wrappers. We should wait on this issue until that is decided (possibly here, and possibly in https://github.com/phetsims/phet-io-wrappers/issues/84).

**Comment by zepumph** [comment]
Tagging @samreid so that he is aware of this, as we move on with implementing "testing" mode with the studio wrapper or other places.

@jonathanolson will you please take a look at the above commit. Does this seem alright to you?

**Comment by samreid** [comment]
> Currently I'm about 90% sure we will end up wanting debug versions of the sim (with assertions left in) in ever phet-io build ever to support phet-io client devs of wrappers.

That sounds right to me, but I'd feel a lot better if we could track down an issue where we confirmed it is OK to publish unminified phet-io sim-side code.  Or perhaps we decided to go with minified, but with assertions not stripped?  I can't remember.

**Comment by zepumph** [comment]
https://github.com/phetsims/phet-io/issues/1149#issuecomment-348319809 and the comment before.

**Comment by samreid** [comment]
The issue @zepumph referenced said we would still like to mangle the code.  The commit @zepumph made creates a mangled version that leaves the assertions in, which is what we want.

**Comment by samreid** [comment]
@jonathanolson can you double check and make sure we are barking up a nice tree?

**Comment by jonathanolson** [comment]
So you're saying that we want to force ANY local build of the phet-io brand to include the debug build? (This is separate from any deployment, since the --debugHTML flag is provided with any deployment).

**Comment by zepumph** [comment]
That's right!

**Comment by jonathanolson** [comment]
This is being done already now (as part of another issue I believe).
